Debian is moving from rpassword 5 to rpassword 6.
This change should be sufficient to work properly.

Author: Daniel Kahn Gillmor <dkg@fifthhorseman.net>

diff --git a/src/lib.rs b/src/lib.rs
index 4199249..5e98cfa 100644
--- a/src/lib.rs
+++ b/src/lib.rs
@@ -234,10 +234,10 @@ impl pam::Converse for PamConvo {
         CString::new(self.login.clone()).map_err(|_| ())
     }
     fn prompt_blind(&mut self, _msg: &CStr) -> Result<CString, ()> {
-        match rpassword::read_password_from_tty(Some(&format!(
+        match rpassword::prompt_password(format!(
             "[{}] password for {}: ",
             self.service, self.login
-        ))) {
+        )) {
             Ok(password) => self.passwd = Some(password),
             Err(_) => {
                 println!("Cannot read from terminal");
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -44,7 +44,7 @@
 version = "1.3"
 
 [dependencies.rpassword]
-version = "5.0"
+version = "6"
 
 [dependencies.syslog]
 version = ">= 6.0"
